### DISCUSSION TITLE
Is there any plan of apply i18n at doc?

### DISCUSSION THREAD

**Discussion by jujumilk3** [apache/airflow#23471] [Ideas] [answered]
Airflow's doc is very useful but it provides english only.
I wanna ask for plan of doc's i18n and if there's plan of it, i wanna join.

**Comment by potiuk** [comment, marked as answer]
No plans. It would be an enormous effort to translate the documentation of ours as it is huge. And even more effort to keep it updated - whatever languages we will support,  currently we rely on the fact that our contributors know English and provide documentation together with each change. If we add i18n, there is a huge risk of slowing down any changes or getting the documentation desynchronized between language (which is worse than not having different languages at all as people might take such an oudated documentation as the "source of truth"). 

I personally think with the size of the project and itsopen-source nature we cannot do better than what modern browsers/cloud providers support i.e. translating the docs automatically in the browser.

I just experimented and switched translation of the docs into my native Polish via Chrome's translate feature - and while the result is not perfect, I seriously doubt we can do any better than that (and certainly if you see how many languages it supports out of the box we certainly can't do any better for all the languages out there). In the translation to Polish (which is one of the most difficult languages in the world for automated translation due to its exceptions, grammar and flex structure) the text is perfectly legible, understandable and while it contains a number of stylistic errors and strange choices for words used, it's quite usable actually.

<img width="1793" alt="Screenshot 2022-05-09 at 12 47 21" src="https://user-images.githubusercontent.com/595491/167395201-fc7be907-b12d-42cf-9830-113d99dedec4.png">
